src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLSPConventions.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.CodeAnalysis.Razor;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor
+{
+    internal static class RazorLSPConventions
+    {
+        public const string RazorLSPContentTypeName = "RazorLSP";
```

Comment:
We typically have constants classes to house this type of information and convention classes to have common utility methods. Any reason in particular this made more sense here?

Reply:
They just seemed to go well together. I can move the consts in this file to a `RazorLSPConstants` if you like that better.